pkg/synthetictests/kubelet.go

```diff
@@ -229,11 +235,29 @@ func formatTimes(times []time.Time) []string {
 	return s
 }
 
-func testNodeUpgradeTransitions(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+func testNodeUpgradeTransitions(events monitorapi.Intervals, kubeClientConfig *rest.Config) []*junitapi.JUnitTestCase {
 	const testName = "[sig-node] nodes should not go unready after being upgraded and go unready only once"
 
 	var buf bytes.Buffer
 	var testCases []*junitapi.JUnitTestCase
+
+	// TODO: This test modification will need to be re-examned and removed if no longer needed, should only effect upgrades to 4.11
+	// The purpose of this is to account for an update in NTO which will force a second restart during upgrades
+	// for a cluster configuration that makes use of TuneD and the stalld service.
+	// Currently this should only occur on realtime kernels
+	// A failure here should not stop the entire test
+	is411Upgrade, err := is411MajorUpgrade(kubeClientConfig)
+	if err != nil {
+		fmt.Fprintf(&buf, "DEBUG: failed to get cluster version, continuing : %v\n", err)
+	}
+	var realtimeNodes = make(map[string]int)
```

Comment:
Suggest just to clarify what's really in there: realtimeNodeConfigReboots

Reply:
Thanks for the suggestion, that's more clear and less ambiguous, updated!